gitops/prepare/prepare.go

```diff
@@ -110,8 +114,13 @@ func ResolveBuildBundle(ctx context.Context, localClient, cli client.Client, nam
 
 // Return true when hacbs configmap exists in the namespace
```

Comment:
nit: might be worth to update the description of the function

Reply:
updated